### DISCUSSION TITLE
Suggestions

### DISCUSSION THREAD

**Discussion by ts-thomas** [nextapps-de/winbox#66] [Ideas]
- snap windows (see magnet.js)
- shortcut minimize all / restore all
- workaround for lost events over iframe (using the message channel)
- maximize + restore icon separately
- improve tiled windows

**Comment by vlrevolution** [comment]
I'm interested in these

**Comment by ordo-n** [comment]
It would be nice if could configure the taskbar button to always be visible like it is in windows. At the moment it's only visible when the window is minimized.
